misc/tutorial/401_AllFeatures.ngdoc

```diff
@@ -90,13 +98,12 @@ All features are enabled to get an idea of performance
           $timeout.cancel(timeout);
           $interval.cancel(sec);
         });
-
       };
-
     }]);
   </file>
   <file name="index.html">
     <div ng-controller="MainCtrl">
+      <button id="filterToggle" type="button" class="btn btn-success" ng-click="toggleFilterRow()">Toggle Filter</button>
```

Comment:
I find myself needing to test this a lot and I frequently see question about how to implement something like this, so I added it to this demo.